<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiMachineConstant</class>
 <widget class="QDialog" name="uiMachineConstant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,5,1" columnstretch="3,1,1,1">
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>总参数</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>机床总长度</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="machine_length"/>
          </item>
          <item row="0" column="3" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>机床立柱间距离</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="column_length"/>
          </item>
          <item row="1" column="3" colspan="2">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>工作台惯量</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="stage_inert"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>滑块惯量</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="slider_inert"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>刚强</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QLineEdit" name="firm"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>机床结构刚度</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QLineEdit" name="struct_firm"/>
          </item>
          <item row="4" column="3" colspan="2">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>um/kN</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3" colspan="2">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>%</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>模具间的参考距离</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>模具安装方式校正</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QLineEdit" name="reference_distance"/>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QLineEdit" name="correct"/>
          </item>
          <item row="6" column="3" colspan="2">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3" colspan="2">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="sure">
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_23">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>机床参数设置</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>轴参数</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5" columnstretch="2,1,1,1,1">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Y轴最大值</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="y_max"/>
          </item>
          <item row="0" column="3" colspan="2">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Y轴最小值</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="y_min"/>
          </item>
          <item row="1" column="3" colspan="2">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>X轴最大值</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLineEdit" name="x_max"/>
          </item>
          <item row="2" column="3" colspan="2">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>X轴最小值</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="x_min"/>
          </item>
          <item row="3" column="3" colspan="2">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>R轴最大值</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QLineEdit" name="r_max"/>
          </item>
          <item row="4" column="3" colspan="2">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Z轴最大值</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QLineEdit" name="z_max"/>
          </item>
          <item row="6" column="3" colspan="2">
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Z轴最小值</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3" colspan="2">
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>R轴最小值</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3" colspan="2">
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QLineEdit" name="r_min"/>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QLineEdit" name="z_min"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
